src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+import nlpaug.augmenter.char as nac
+
+
+def keyboard_typo(
+    texts: List[str],
+    **kwargs,
+) -> List[str]:
+    '''Generate keyboard typo perturbed texts for augmentation.
+
+    Args:
+        texts: List of texts to be augmented.
+        **kwargs: Arbitrary keyword arguments, which will be passed to
+            :class:`nlpaug.augmenter.char.keyboard.KeyboardAug`. See the
+            documentation for more details.
```

Comment:
Could you add the link to their readthedocs?
https://nlpaug.readthedocs.io/en/latest/augmenter/char/keyboard.html